rails/app/models/theme.rb

```diff
@@ -1,9 +1,19 @@
 class Theme < ApplicationRecord
+  include MapConfigurable
   has_one_attached :background_img
   has_many_attached :sponsor_logos
+  after_initialize :set_map_defaults
```

Comment:
i think this is an interesting way to avoid worrying about pre-existing values for now, and think it works so good for now.

It feels a little funny long term, but i might just be more attached to db defaults.